spec/views/companies/index.html.erb_spec.rb

```diff
@@ -2,6 +2,28 @@
 
 require 'rails_helper'
 
-RSpec.describe 'companies/index.html.erb', type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'companies/index.html.erb rendering locals in a partial', type: :view do
```

Comment:
In `describe` you only described what unit is under test.

```suggestion
RSpec.describe 'companies/index.html.erb', type: :view do
```